src/analysis/Environments/Environments.test.ts

```diff
@@ -81,8 +78,12 @@ const getMockLeoDiskProvider = (overrides?: Partial<LeoDiskProvider>): LeoDiskPr
 };
 
 const getEnvironmentsProps = (propsOverrides?: Partial<EnvironmentsProps>): EnvironmentsProps => {
-  asMockedFn(mockPermissions.canDeleteDisk).mockReturnValue(true);
-  asMockedFn(mockPermissions.canPauseResource).mockReturnValue(true);
+  const mockPermissions: PermissionsAndStateProvider = {
+    canDeleteDisk: jest.fn().mockReturnValue(true),
```

Comment:
unless the type-flow for jest.fn() has improved, this pattern is not type safe... try mockReturnValue('true') which should fail type safety.

I would revert this to some form that uses asMockedFn to ensure type safety.

Reply:
Hm, I changed due to the errors I was getting throughout the test suite when run:
```(0 , _testUtils.asMockedFn)(...).mockReturnValue is not a function
TypeError: (0 , _testUtils.asMockedFn)(...).mockReturnValue is not a function```
Is there a specific reason that I'm seeing these errors when using `asMockedFn()`?